src/map/lua/lua_baseentity.cpp

```diff
@@ -803,6 +803,8 @@ void CLuaBaseEntity::StartEventHelper(int32 EventID, sol::variadic_args va, EVEN
         return;
     }
 
+    PChar->StatusEffectContainer->DelStatusEffect(EFFECT_BOOST);
```

Comment:
So, I would suggest we confirm that teleports remove it as well before sticking it here.  If they don't then it should most likely move to a condition in ParseEvent.

Reply:
sec I can check that